### ISSUE TITLE
Enhanced user info page

### ISSUE THREAD

**Issue by eutampieri** [algorithm-ninja/cmsocial#18]
I think that a user should be able to edit his school too. Now he's able only to change his password.

**Comment by eutampieri** [comment]
I need this because I want to see users of my school but not everybody has set the school during registration so, if you add this feature, people is able to add school info.

**Comment by wil93** [comment]
Yes this should be done, ideally a user should be able to change password / username / first name / last name / school.

**Comment by eutampieri** [comment]
If you want I can try to help with my basic Python knowledge...

**Comment by wil93** [comment]
Users can now change school information. Still missing:

- [ ] Changing username
- [x] Changing first name / last name
- [ ] Changing email
